src/row.ts

```diff
@@ -156,13 +157,13 @@ export class RowError extends Error {
  */
 export class Row {
   bigtable: Bigtable;
-  table: Table;
+  table: TabularApiService;
   id: string;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   data: any;
   key?: string;
   metadata?: {};
-  constructor(table: Table, key: string) {
+  constructor(table: TabularApiService, key: string) {
```

Comment:
Is this meant to be `TabularAPISurface` or `TabularAPIService`?

Reply:
Good catch. It should be TabularAPISurface. I changed this everywhere now.